sup3r/preprocessing/data_handling/exo_extraction.py

```diff
@@ -179,8 +185,7 @@ def __init__(self,
     def source_data(self):
         """Get the 1D array of source data from the exo_source_h5"""
 
-    def get_cache_file(self, feature, s_enhance, t_enhance, s_agg_factor,
-                       t_agg_factor):
+    def get_cache_file(self, feature, s_enhance, t_enhance, t_agg_factor):
```

Comment:
Yeah I actually removed the s_agg_factor from the whole module altogether because for the topo aggregation "factor" it's now determined by the actual pixel mapping.